modular_skyrat/modules/aesthetics/airlock/code/airlock.dm

```diff
@@ -560,12 +567,15 @@
 //SKYRAT EDIT ADDITION BEGIN - AESTHETICS
 #undef AIRLOCK_LIGHT_POWER
 #undef AIRLOCK_LIGHT_RANGE
-
+#undef AIRLOCK_LIGHT_ENGINEERING
+#undef AIRLOCK_LIGHT_SECURITY
+#undef AIRLOCK_ENGINEERING_LIGHT_COLOR
 #undef AIRLOCK_POWERON_LIGHT_COLOR
 #undef AIRLOCK_BOLTS_LIGHT_COLOR
 #undef AIRLOCK_ACCESS_LIGHT_COLOR
 #undef AIRLOCK_EMERGENCY_LIGHT_COLOR
 #undef AIRLOCK_DENY_LIGHT_COLOR
+#undef AIRLOCK_SECURITY_LIGHT_COLOR
 //SKYRAT EDIT END
```

Comment:
While you're here, please remove the Skyrat Edit comments, it's literally a modular file.